mojo-ac/.htaccess

```diff
@@ -0,0 +1,10 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+
+RewriteRule ^$ https://mojoac.wordpress.com/ [R=302,L]
+
+RewriteRule ^Blog$ https://mojoac.wordpress.com/Blog [R=302,L]
+RewriteRule ^Blog/$ https://mojoac.wordpress.com/Blog  [R=302,L]
+RewriteRule ^Blog/(.*)$ https://mojoac.wordpress.com/Blog#1  [NE,R=302,L]
```

Comment:
Did you mean to use that pattern for wordpress style paths?  Perhaps this should be:
```apache
RewriteRule ^Blog/(.*)$ https://mojoac.wordpress.com/$1  [NE,R=302,L]
```